operator/cmd/mesh/manifest-generate.go

```diff
@@ -141,7 +141,7 @@ func GenManifests(inFilename []string, setOverlayYAML string, force bool,
 		return nil, nil, err
 	}
 	if err := cp.Run(); err != nil {
-		return nil, nil, fmt.Errorf("failed to create Istio control plane with spec: \n%v\nerror: %s", mergedIOPS, err)
```

Comment:
I think this error is quite useful for debugging in-cluster operator (controller). Should we keep it a little longer?

Reply:
the problem is it outputs a huge, unformatted blob of text and obscures the real error. we should have something here, but perhaps through --logtostderr, and with a formatted output?

Reply:
Error: failed to create Istio control plane with spec: 
profile:"/usr/local/google/home/mostrowski/go/src/istio.io/istio/operator/data/profiles/default.yaml" install_package_path:"/usr/local/google/home/mostrowski/go/src/istio.io/istio/manifests2" hub:"gcr.io/istio-testing" tag:latest components:<base:<enabled:<BoolValue:<value:true > > > pilot:<enabled:<BoolValue:<value:true > > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAME" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.name" > > > env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > readiness_probe:<httpGet:<path:"/ready" port:8080 > initialDelaySeconds:5 timeoutSeconds:5 periodSeconds:5 > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > policy:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-policy" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > telemetry:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > env:<name:"GOMAXPROCS" value:"6" > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-telemetry" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > replica_count:1 resources:<limits:<key:"cpu" value:"4800m" > limits:<key:"memory" value:"4G" > requests:<key:"cpu" value:"1000m" > requests:<key:"memory" value:"1G" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > citadel:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > galley:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<replica_count:1 resources:<requests:<key:"cpu" value:"100m" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > cni:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> > ingress_gateways:<enabled:<BoolValue:<value:true > > name:"istio-ingressgateway" tag:<nil> k8s:<hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-ingressgateway" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > resources:<limits:<key:"cpu" value:"2000m" > limits:<key:"memory" value:"1024Mi" > requests:<key:"cpu" value:"100m" > requests:<key:"memory" value:"128Mi" > > strategy:<rollingUpdate:<ma
...

Reply:
sounds good if we change into log only.